modules/account-map/README.md

```diff
@@ -14,9 +14,6 @@ components:
     account-map:
       vars:
         enabled: true
-        # If this is true, profile must be provided instead of role_arn in each backend.tf.json
-        # If this is false, role_arn must be provided instead of profile in each backend.tf.json
-        profiles_enabled: false
```

Comment:
Why is this removed ?

Reply:
Restored it immediately and forgot to push